pkg/driver/deployment/knative.go

```diff
@@ -83,11 +87,63 @@ func knativeDeploySingleFunction(function *common.Function, yamlPath string, isP
 		// second, then round to an integer as that is what the knative config expects
 	}
 
+	// Default is the function name, but if the function is a vSwarm function, the proxy name is used
+	proxyName := function.Name
+
+	if vSwarm {
+		// Read and unmarshal the mapper output file into a map
+
+		mapperFile, err := os.ReadFile("tools/mapper/output.json")
```

Comment:
Can you make it without hard-coding the path to the file? For example, it might be more useful to move mapper's output to the trace directory instead of using the global file for the mappings.